pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -121,38 +161,14 @@ func (c *PeriodicBackupController) sync(ctx context.Context, _ factory.SyncConte
 		}
 	}
 
-	if defaultFound {
-		mirrorPods, err := c.podLister.List(labels.Set{"app": "etcd"}.AsSelector())
+	if !defaultFound {
+		err = c.kubeClient.AppsV1().DaemonSets(operatorclient.TargetNamespace).Delete(ctx, backupServerDaemonSet, v1.DeleteOptions{})
 		if err != nil {
-			return fmt.Errorf("PeriodicBackupController could not list etcd pods: %w", err)
-		}
-
-		var terminationReasons []string
-		for _, p := range mirrorPods {
-			for _, cStatus := range p.Status.ContainerStatuses {
-				if cStatus.Name == etcdBackupServerContainerName {
-					// TODO we can also try different cStatus.State.Terminated.ExitCode
-					terminationReasons = append(terminationReasons, fmt.Sprintf("container %s within pod %s has been terminated: %s", etcdBackupServerContainerName, p.Name, cStatus.State.Terminated.Message))
-				}
-			}
-		}
-
-		if len(terminationReasons) > 0 {
-			_, _, updateErr := v1helpers.UpdateStatus(ctx, c.operatorClient, v1helpers.UpdateConditionFn(operatorv1.OperatorCondition{
-				Type:    "PeriodicBackupControllerDegraded",
-				Status:  operatorv1.ConditionTrue,
-				Reason:  "Error",
-				Message: fmt.Sprintf("found default backup errors: %s", strings.Join(terminationReasons, " ,")),
-			}))
-			if updateErr != nil {
-				klog.V(4).Infof("PeriodicBackupController error during default backup UpdateStatus: %v", err)
+			if !apierrors.IsNotFound(err) {
+				return fmt.Errorf("PeriodicBackupController could not delete [defaultBackupDeployment]: %w", err)
 			}
-
-			return nil
 		}
-	} else {
-		// disable etcd-backup-server
-		c.backupVarGetter.SetBackupSpec(nil)
+		klog.V(4).Infof("PeriodicBackupController deleted DaemonSet [%v] successfully", backupServerDaemonSet)
```

Comment:
isn't this going to log every time there is no default CRD found?

Reply:
good catch 👍🏽 

fixing ...